plugin.yml

```diff
@@ -1,22 +1,19 @@
 ---
 name: PocketRadio
 main: xenialdan\PocketRadio\Loader
-version: 4.2.2
-api: [3.10.1,3.11.0,3.12.0,3.13.0,3.14.0,3.15.0,3.16.0,3.17.0]
-mcpe-protocol: [388,389,390,391,392,393,394,395,396,397,398,399,400,401,402,403,404,405,406,407,408,409,410,411,412,413,414,415,416,417,418,419,420,421,422]
```

Comment:
Do not remove this.

Reply:
I talked about protocol

Reply:
> I talked about protocol

Protocol is no longer required according to the latest Poggit rule.